code/datums/flock/flock.dm

```diff
@@ -688,6 +688,22 @@ var/flock_signal_unleashed = FALSE
 	/obj/machinery/computer = /obj/flock_structure/compute,
 	/obj/machinery/networked/teleconsole = /obj/flock_structure/compute,
 	/obj/machinery/networked/mainframe = /obj/flock_structure/compute/mainframe,
+	/obj/machinery/vending = /obj/flock_structure/fabricator,
+	/obj/machinery/manufacturer = /obj/flock_structure/fabricator,
+	/obj/submachine/seed_vendor = /obj/flock_structure/fabricator,
+	/obj/machinery/dispenser = /obj/flock_structure/fabricator,
+	/obj/machinery/disposal_pipedispenser = /obj/flock_structure/fabricator,
+	/obj/machinery/chem_dispenser = /obj/flock_structure/fabricator,
+	/obj/machinery/chemicompiler_stationary = /obj/flock_structure/fabricator,
+	/obj/reagent_dispensers/foamtank = /obj/flock_structure/fabricator,
+	/obj/reagent_dispensers/watertank = /obj/flock_structure/fabricator,
+	/obj/reagent_dispensers/fueltank = /obj/flock_structure/fabricator,
+	/obj/reagent_dispensers/heliumtank = /obj/flock_structure/fabricator,
+	/obj/reagent_dispensers/compostbin = /obj/flock_structure/fabricator,
+	/obj/reagent_dispensers/beerkeg = /obj/flock_structure/fabricator,
+	/obj/machinery/portable_atmospherics/canister = /obj/flock_structure/fabricator,
```

Comment:
Flock currently benefit heavily from using various gas canisters, probably a bad idea for drones to automatically convert them.

Reply:
Idk, I think in a way it's also up to player choice too

Reply:
Well it's not player choice if the drones do it automatically? You'd have to make sure you never leave your drones around gas canisters or they'll eat them.

Reply:
Can change, but should Flockdrones be doing that? Nothing stopping people from doing it, but idk about Flockdrones relying on station stuff rather than their own structures

Reply:
More of a high level design question, but personally I like flock being able to do stuff that's not strictly confined to their abilities like playing with atmos or rogueing the AI. Makes for more gameplay variety.

Reply:
I'll remove it for the moment, can always add back later